clj-refactor.el

```diff
@@ -2350,7 +2350,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-promote-function
   (= (point) (cljr--point-after 'end-of-buffer)))
 
 (defun cljr--find-symbol-sync (symbol ns)
-  (let* ((filename (buffer-file-name))
+  (let* ((filename (funcall cider-to-nrepl-filename-function (buffer-file-name)))
```

Comment:
You can delete all occurrences of `funcall`.  If you know the name of the function you can call it like this:

`(cider-to-nrepl-filename-function arg-goes-here)`